lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -18,13 +33,32 @@ def self.run(params)
           violations.each do |lang, diff|
             UI.error "Inconsistencies found between '#{params[:base_lang]}' and '#{lang}':\n\n#{diff}\n"
           end
-          if params[:abort_on_violations] && !violations.empty?
-            UI.abort_with_message!('Inconsistencies found during Localization linting. Aborting.')
-          end
-          
+
           violations
         end
 
+        RETRY_MESSAGE = <<~MSG
+          Inconsistencies found during Localization linting.
+          You need to fix them before continuing. From this point on, you should:
+
+          - Either cancel this lane (reply 'No' below), then work with polyglots in #i18n
+            to fix those directly in GlotPress – by rejecting the inconsistent
+            translations, or by submitting a fixed copy. Then run the lane again.
+
+            This is the recommended way to go, as it will fix the issues at their source.
+
+          - Or manually edit the `Localizable.strings` files to fix the inconsistencies
+            locally, commit them, then reply 'Yes' below to re-lint and validate that all
+            inconsistencies have been fixed locally so you can continue with the build.
+
+            This is only a workaround to allow you to submit a build if translators are
+            not available to help you fix the issues in GlotPress in time. You will still
+            need to let the translators know that they will need to fix those copies
+            at some point before the next build to fix the root issue.
+
+          Did you fix the .strings files locally and want to lint them again?
```

Comment:
```suggestion
          Did you fix the `.strings` files locally and want to lint them again?
```

Suggesting this because a few line above we use backticks for `Localizable.string`. 

Alternatively, we could remove the backticks above, too. Either way, imho it'd be good to be consistent.